packages/starknet-snap/src/estimateFee.ts

```diff
@@ -25,6 +25,7 @@ export async function estimateFee(params: ApiParams) {
     const contractCallData = getCallDataArray(requestParamsObj.contractCallData);
     const senderAddress = requestParamsObj.senderAddress;
     const network = getNetworkFromChainId(state, requestParamsObj.chainId);
+    const transactionVersion = requestParamsObj.transactionVersion ?? TRANSACTION_VERSION;
```

Comment:
minor.
may be we dont need default value? as estimateFeeUtil and estimateFeeBulk already handled?